package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ravendb",
-  "version": "4.0.0-beta",
+  "version": "4.0.0-rc-40019",
```

Comment:
just FYI we're not going to match server version with client version. It's going to be 4.0.0-rc1